[2024-10-23T00:31:28.630+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-23T00:31:28.691+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: aula_4_1_get_stocks_dag.AAPL scheduled__2022-01-28T00:00:00+00:00 [queued]>
[2024-10-23T00:31:28.732+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: aula_4_1_get_stocks_dag.AAPL scheduled__2022-01-28T00:00:00+00:00 [queued]>
[2024-10-23T00:31:28.732+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-23T00:31:28.759+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): AAPL> on 2022-01-28 00:00:00+00:00
[2024-10-23T00:31:28.783+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=314) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-23T00:31:28.789+0000] {standard_task_runner.py:72} INFO - Started process 364 to run task
[2024-10-23T00:31:28.811+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'aula_4_1_get_stocks_dag', 'AAPL', 'scheduled__2022-01-28T00:00:00+00:00', '--job-id', '336', '--raw', '--subdir', 'DAGS_FOLDER/aula_4_1_get_stocks.py', '--cfg-path', '/tmp/tmpghmnwr1q']
[2024-10-23T00:31:28.819+0000] {standard_task_runner.py:105} INFO - Job 336: Subtask AAPL
[2024-10-23T00:31:28.916+0000] {task_command.py:467} INFO - Running <TaskInstance: aula_4_1_get_stocks_dag.AAPL scheduled__2022-01-28T00:00:00+00:00 [running]> on host 375c06cd6615
[2024-10-23T00:31:29.224+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='aula_4_1_get_stocks_dag' AIRFLOW_CTX_TASK_ID='AAPL' AIRFLOW_CTX_EXECUTION_DATE='2022-01-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-01-28T00:00:00+00:00'
[2024-10-23T00:31:29.229+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-23T00:31:29.278+0000] {cache.py:130} INFO - Failed to create TzCache, reason: Error creating TzCache folder: '/home/***/.cache/py-yfinance' reason: [Errno 13] Permission denied: '/home/***/.cache/py-yfinance'. TzCache will not be used. Tip: You can direct cache to use a different location with 'set_tz_cache_location(mylocation)'
[2024-10-23T00:31:29.280+0000] {cache.py:336} INFO - Failed to create CookieCache, reason: Error creating CookieCache folder: '/home/***/.cache/py-yfinance' reason: [Errno 13] Permission denied: '/home/***/.cache/py-yfinance'. CookieCache will not be used. Tip: You can direct cache to use a different location with 'set_tz_cache_location(mylocation)'
[2024-10-23T00:32:43.254+0000] {history.py:245} ERROR - $AAPL: possibly delisted; no price data found  (1h 2022-01-27 -> 2022-01-28)
[2024-10-23T00:32:43.580+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-23T00:32:43.748+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-23T00:32:43.749+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=aula_4_1_get_stocks_dag, task_id=AAPL, run_id=scheduled__2022-01-28T00:00:00+00:00, execution_date=20220128T000000, start_date=20241023T003128, end_date=20241023T003243
[2024-10-23T00:32:43.779+0000] {job.py:229} INFO - Heartbeat recovered after 75.34 seconds
[2024-10-23T00:32:43.996+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-23T00:32:44.040+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-23T00:32:44.060+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
